sdk/tests/conformance/reading/00_test_list.txt

```diff
@@ -1,3 +1,3 @@
 read-pixels-pack-alignment.html
 read-pixels-test.html
-
+--min-version 1.0.4 fbo-remains-unchanged-after-read-pixels.html
```

Comment:
Sorry, late reply for vacation. I had ordered these files according alphabetical order.